frequency-chain/testnet-faucet
# Feat/replace font remove dropdown

## Goal

The goal of this PR is to replace all fonts with Poppins (a Google font) and to remove dropdowns for choosing the network and chain since we have only one faucet.  We will never support a faucet for another network besides Frequency and if we support another Frequency chain we can add a selection for that later.

Closes #20 
Closes #21 

## Discussion
I chose to pull the typeface from Google servers instead of checking it into our code.

## Checklist


## How To Test the Changes

1. Clone the pr branch
2. Launch a local frequency chain instance with `make start`
3. Launch the faucet server:
```shell
git checkout feat/replace-font-remove-dropdown
yarn install --frozen-lockfile
yarn dev
```
4.  Launch the faucet client:
```shell
cd client
yarn install --frozen-lockfile
yarn dev -- --open
```
5. Verify that the font has changed to Poppins
6. Verify that there are no dropdowns to select a network or chain
7. Verify that you can successfully get token from the faucet (local chain)
